README.md

```diff
@@ -62,6 +62,12 @@ With these valuable insights, the customer service representative can make concr
 
 - **Health Care** - Medical professionals are working to ensure there is enough hospital capacity Covid patients. They use a variety of data about infected patients to identify those likely to require hospitalization. The dashboard can also guide their treatments, to help reduce the number of hospitalizations. 
 
+- **Education** - Student attrition rates are a key performance indicator for any higher education provider and have a significant
+impact on students. Understanding what causes student attrition (ie financial status, academic performance, or student wellbeing)
+is a complex challenge which is often specific to individual student circumstances. The RAI dashboard can help higher education decision makers better understand student attrition while informingthe creation of intervents to improve student support services and foster student engagment and belonging.
```

Comment:
"while informingthe creation of intervents to improve student support services"
can this be rephrased?  the text looks confusing.

Reply:
Yes, will rephrase.